src/components/multiview/VideoSelector.vue

```diff
@@ -1,36 +1,32 @@
 <template>
     <!-- Vertical: -->
-    <v-card class="pa-3 vertical-video-selector" v-if="!horizontal">
-        <v-row>
-            <v-col cols="12" sm="4" md="3" lg="2">
+    <v-card class="pa-3" v-if="!horizontal">
+        <v-row class="flex-column" :class="{ 'flex-nowrap': $vuetify.breakpoint.xs }" style="height: 80vh">
```

Comment:
Minor change but maybe `flex-nowrap flex-sm-wrap`, keeping the layout change in CSS instead of JS.

Reply:
Tested with CSS solution and works.